src/api/mutations/accommodations/useDeleteAccommodation.ts

```diff
@@ -21,7 +24,7 @@ export const useDeleteAccommodation = () => {
       await queryClient.invalidateQueries({
         queryKey: [QUERY_KEYS.query.hostedAccommodation, accommodationId],
       });
-      toast.success('Accommodation deleted successfully');
+      toast.success(`${t(ToastMessages.SuccessAccDel)}`);
```

Comment:
It's better not to cut message names

`SuccessAccountDeleted`

Reply:
sorry, it is orignally SuccessAccommodationDeleted. it was my mistake, and fixed